src/core/solver/wasm_solver.ts

```diff
@@ -89,7 +89,10 @@ export class WASMSolver extends ConstraintSolver {
     attr.map[attr.name] = value;
   }
 
-  /** Add a linear constraint */
+  /** Add a linear constraint
```

Comment:
Should this be on the next line, since you have the expanded jsdoc syntax?

Reply:
Enabled check-multiline-start for jsdoc-format